### DISCUSSION TITLE
Migrations  / modifying relations

### DISCUSSION THREAD

**Discussion by alexgraul** [cozodb/cozo#270] [Q&A]
Hey folks, how is everyone handling making modifications to existing relations? Simply a case of create a new relation / copy the data / delete the old one / rename? While viable this seems very expensive, are there any other routes?

**Comment by creatorrr** [comment]
I created a library called cozo-migrate that's on pypi. (You can use it outside of python projects also)

https://github.com/julep-ai/cozo-migrate
